src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -277,10 +277,31 @@ void SetParameter()
          Aux_Error( ERROR_INFO, "Incorrect parameter %s = %d !!\n", "CCSN_Is_PostBounce", CCSN_Is_PostBounce );
    }
 
-// CCSN_Min_AngRes must be larger than CCSN_Min_AngRes
-   if ( CCSN_Min_AngRes > 0.0  &&  CCSN_Max_AngRes > 0.0 )
-      if ( CCSN_Min_AngRes <= CCSN_Max_AngRes )
-         Aux_Error( ERROR_INFO, "%s = %d must be larger than %s = %d  !!\n", "CCSN_Min_AngRes", CCSN_Min_AngRes, "CCSN_Max_AngRes", CCSN_Max_AngRes );
+// check OPT__FLAG_REGION is enabled for CCSN_AngRes_Min/Max and convert degree to radian
+   const double DEG2RAD = M_PI/180.0;
+   if ( CCSN_AngRes_Min > 0.0 ) {
+      CCSN_AngRes_Min *= DEG2RAD;
+      if ( !OPT__FLAG_REGION )
+         Aux_Error( ERROR_INFO, "%s is disabled for %s = %13.7e!!\n", "OPT__FLAG_REGION", "CCSN_AngRes_Min", CCSN_AngRes_Min );
+   }
+
+   if ( CCSN_AngRes_Max > 0.0 ) {
+      CCSN_AngRes_Max *= DEG2RAD;
+      if ( !OPT__FLAG_REGION )
+         Aux_Error( ERROR_INFO, "%s is disabled for %s = %13.7e!!\n", "OPT__FLAG_REGION", "CCSN_AngRes_Max", CCSN_AngRes_Max );
+   }
+
+   if (  ( CCSN_AngRes_Min > 0.0  ||  CCSN_AngRes_Max > 0.0 )  &&  !OPT__FLAG_REGION  )
+      Aux_Error( ERROR_INFO, "%s is disabled for %s = %13.7e, %s = %13.7e!!\n",
+                 "OPT__FLAG_REGION", "CCSN_AngRes_Min", CCSN_AngRes_Min, "CCSN_AngRes_Max", CCSN_AngRes_Max );
+
+   if ( CCSN_AngRes_Min > 0.0  &&  CCSN_AngRes_Max > 0.0 )
+   {
+      if ( CCSN_AngRes_Min <= CCSN_AngRes_Max )
+         Aux_Error( ERROR_INFO, "%s = %13.7e must be larger than %s = %13.7e  !!\n", "CCSN_AngRes_Min", CCSN_AngRes_Min, "CCSN_AngRes_Max", CCSN_AngRes_Max );
+      if ( CCSN_AngRes_Min < 2.0 * CCSN_AngRes_Max )
+         Aux_Message( stdout, "WARNING : CCSN_Min_AngRes < 2.0 * CCSN_Max_AngRes: CCSN_Max_AngRes might not be strictly followed!!\n" );
```

Comment:
correct `CCSN_Min/Max_AngRes`

Reply:
There is still one that hasn't been corrected.

Reply:
Sorry for the missing, and thanks for the careful review. I corrected this.